src/sentry/integrations/cursor/webhooks/handler.py

```diff
@@ -90,15 +91,20 @@ def _get_cursor_integration_secret(self, organization_id: int) -> str | None:
             )
             return None
 
-        integration = integrations[0]
-        if "webhook_secret" in integration.metadata:
-            return integration.metadata["webhook_secret"]
-        else:
+        installation = integrations[0].get_installation(organization_id)
```

Comment:
Are we able to make assurances that an organization only has a single installation at any given time? This seems like a weak guarantee of that.

Reply:
This is actually caught just above the diff for the PR, so I think it's alright 👍

Reply:
it should error above if there are more than one, but good call we shouldn't even let them install more than 1